steps/pipeline-tests-jest.yaml

```diff
@@ -0,0 +1,42 @@
+parameters:
+- name: environment
+  default: ''
+- name: domain_prefix
+  default: ''
+- name: test_file
+  default: ''
+- name: workingDirectory
+  default: ''
+- name: junit_output_dir
+  default: ''
+- name: test_title
+  default: ''
+- name: root_domain
+  default: platform.hmcts.net
+- name: test_url
+  default: ''
+
+steps:
+- task: Bash@3
+  displayName: Run pipeline tests
+  inputs:
+    targetType: 'inline'
+    script: |
+      mkdir -p ${{ parameters.junit_output_dir }}
+      yarn
+      yarn test
+    workingDirectory: '${{ parameters.workingDirectory }}'
+    failOnStderr: false
+  env:
+    JEST_JUNIT_OUTPUT_DIR: ${{ parameters.junit_output_dir }}
+    JEST_JUNIT_OUTPUT_NAME: report_$(system.jobId).xml
+    TEST_URL: ${{ parameters.test_url }}
```

Comment:
the test url shouldn't be a parameter it should be generated by passing something like: `app_name` or `service_name` parameter I think.